tests/src/Kernel/Extractor/TwigExtractorTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Drupal\potion\tests\Kernel\Extractor;
+
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\potion\Extractor\TwigExtractor;
+
+/**
+ * Tests Twig Extractor
+ *
+ * @group potion
+ */
+class TwigExtractorTest extends KernelTestBase  {
+
+  public static $modules = [
+    'locale',
+    'language',
+    'potion',
+    'potion_test',
+  ];
+
+  /**
+   * @var \Twig_Environment
+   *  Twig environment
+   */
+  private $twig;
+
+  protected function setUp() {
+    parent::setUp();
+    $this->twig = $this->container->get('twig');
+  }
+
+  public function testExtract() {
+    $expected = [
+      'foo.t',
+      //'foo.trans',
```

Comment:
why commented ?

Reply:
Shouldn’t be. It was when I debugged the tests to be sure it will fail